json/tbd_z5.json

```diff
@@ -0,0 +1,12 @@
+{
+  "schema": "https://github.com/rd2/tbd/blob/master/tbd.schema.json",
+  "description": "testing TBD JSON for 5Zone (plenum)",
+  "psis": [{
+      "id": "salk",
+      "ceiling": 0.1
+  }],
+  "spaces": [{
+    "id": "PLENUM-1",
+    "psi": "salk"
```

Comment:
Somewhat hypothetical experiment: A "Salk Institute"-like plenum floor case (where technical staff walk around on). Postulating here that the _structural_ plenum floor requires perimeter brackets along exterior walls (i.e. thermal bridges of 0.1 W/K per linear meter).